CMakeLists.txt

```diff
@@ -12,7 +12,10 @@ set(SC_MACHINE_THIRDPARTY_PATH "${SC_MACHINE_ROOT}/thirdparty")
 option (SC_AUTO_TEST "Flag to build for automation testing" OFF)
 option (SC_KPM_SCP "Flag to build SCP module" OFF)
 option (SC_BUILD_SCTP "Flag to turn on/off sctp protocol support" OFF)
-option (SC_BUILD_TESTS "Flag to build unit tests" OFF)
```

Comment:
We have no task or issue for these changes, but this just removing of duplicate flag from previous PR (with gtest). Also benchmarks will depend on it. So this is a part of this PR.

Reply:
No, your change is not just removing SC_BUILD_TEST. In this PR you propose to use BUILD_TESTING instead of SC_BUILD_TEST. I'm not against. But your change enable build testing by default because BUILD_TESTING is ON by default(https://cmake.org/cmake/help/latest/module/CTest.html).
Before this change, build testing was not enabled by default. This is significant change of system build and it is debatable.

Reply:
Discussed. Need to create issue for using BUILD_TESTING and resolve it before release 0.7.0